src/components/Modals/SetAclPermissionsModal.jsx

```diff
@@ -0,0 +1,213 @@
+// React Imports
+import React, { useContext, useState } from 'react';
+// Custom Hook Imports
+import { useSession, useStatusNotification } from '@hooks';
+// Material UI Imports
+import Button from '@mui/material/Button';
+import ClearIcon from '@mui/icons-material/Clear';
+import Dialog from '@mui/material/Dialog';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import SettingsIcon from '@mui/icons-material/Settings';
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+// Utility Imports
+import { runNotification, setDocAclPermission, setDocContainerAclPermission } from '@utils';
+// Context Imports
+import { SignedInUserContext } from '@contexts';
+// Component Imports
+import { DocumentSelection, FormSection } from '../Form';
+
+/**
+ * @typedef {import("../../typedefs.js").setAclPermissionsModalProps} setAclPermissionsModalProps
+ */
+
+/**
+ * SetPermissionsModal Component - Modal component that generates the form for
+ * setting ACL permissions to another user's documents, or document container
+ * in their Solid Pod via Solid Session.
+ *
+ * @memberof Modals
+ * @name SetAclPermissionsModal
+ * @param {setAclPermissionsModalProps} Props - Props for SetAclPermsDocContainerModal component
+ * @returns {React.JSX.Element} The SetAclPermsDocContainerModal Component
+ */
+
```

Comment:
If my understanding is correct, we want to remove the space in between JSDocs and their related functions

Reply:
Yes, that is correct. It's so that the JSDoc comment can associate itself with the thing directly below it. The space between them in Line 37 could be removed.

Reply:
Cool, it's been removed